docs/source/services/timelock_service/service-management.rst

```diff
@@ -33,7 +33,8 @@ less than the fresh timestamp. This may be obtained through the fresh timestamp
 
 .. code:: bash
 
-      curl -XPOST localhost:8080/timelock/api/old_namespace/timelock/fresh-timestamp
+    curl -XPOST https://localhost:8421/timelock/api/tl/ts/old_namespace \
+      --data '{"numTimestamps": 1}' -H "Authorization: Bearer q" -H "Content-Type: application/json"
 
 A cluster of TimeLock servers elects a single leader, so if you contact a node that is not the leader you'll receive a
 ``NotCurrentLeaderException`` and an HTTP 503. In this case, try the same request on other nodes until you find the
```

Comment:
For this endpoint I received a 308, not 503

Reply:
Ah, good catch. Yeah, I'll edit this: it can be either